spec/comparators/regexp_comparator_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../spec_helper'
+
+describe RegexpComparator do
+  let(:comparator) { RegexpComparator.new('/Foo Bar/') }
```

Comment:
The `/`'s are not actually part of the regexp, but just the ruby syntax for creating them. t 

I think this test should look like this: 

```ruby
let(:comparator) { RegexpComparator.new('Foo Bar') }
```